config/configv2.go

```diff
@@ -407,50 +409,62 @@ func loadPipelines(v *viper.Viper) (configmodels.Pipelines, error) {
 	return pipelines, nil
 }
 
-func validateConfig(cfg *configmodels.Config) error {
+func validateConfig(cfg *configmodels.Config, logger *zap.Logger) error {
 	// This function performs basic validation of configuration. There may be more subtle
 	// invalid cases that we currently don't check for but which we may want to add in
 	// the future (e.g. disallowing receiving and exporting on the same endpoint).
 
-	if err := validatePipelines(cfg); err != nil {
+	if err := validatePipelines(cfg, logger); err != nil {
 		return err
 	}
 
+	validateReceivers(cfg)
```

Comment:
q.: isn't the pipeline already validating those?

Reply:
No, pipeline validates the list of receivers inside the pipeline. That is different from the top-level receivers.